pkg/kp/kv.go

```diff
@@ -96,7 +134,7 @@ func (c consulStore) PutHealth(res WatchResult) (time.Time, time.Duration, error
 		retDur = writeMeta.RequestTime
 	}
 	if err != nil {
-		return t, retDur, KVError{Op: "put", Key: key, UnsafeError: err}
+		return t, retDur, NewKVError("put", key, err)
```

Comment:
Any particular reason for only converting this one `KVError` to the new format?  Any problems with me doing the same for the rest?